src/actions/websocket.ts

```diff
@@ -611,11 +616,8 @@ function handleChannelDeletedEvent(msg: WebSocketMessage) {
                 }
                 EventEmitter.emit(General.DEFAULT_CHANNEL, '');
             }
-
```

Comment:
What is the purpose of change the code from here to above?

Reply:
Basically we were only dispatching if the channel being removed was part of the current team, so if a channel from another team was removed we were not updating the store and on previous servers that do not return the archived channels then this removed channel was never archived.